tests/unit/views/Curators/Curator.spec.js

```diff
@@ -0,0 +1,229 @@
+import { shallowMount, createLocalVue } from "@vue/test-utils"
+import VueRouter from "vue-router"
+import Vuex from "vuex"
+import sinon from "sinon"
+import Client from "@/components/Client/RESTClient.js"
+import GraphClient from "@/components/GraphClient/GraphClient.js"
+import usersStore from "@/store/users";
+import Curator from "@/views/Curators/Curator.vue"
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+usersStore.state.user = function(){ return {
+    isLoggedIn: true,
+    credentials: {token: 123, username: 123}
+}};
+const $store = new Vuex.Store({
+    modules: {
+        users: usersStore
+    },
+});
+
+const router = new VueRouter();
+const $router = { push: jest.fn() };
+
+describe("Curator.vue", () => {
+  let wrapper;
+  let restStub;
+  let graphStub;
+
+  beforeAll( async (done) => {
+     restStub = await sinon.stub(Client.prototype, "executeQuery").returns({
```

Comment:
This is not an async call.

```suggestion
     restStub = sinon.stub(Client.prototype, "executeQuery").returns({
```